Userland/Libraries/LibJS/Runtime/VM.cpp

```diff
@@ -630,18 +620,9 @@ void VM::run_queued_promise_jobs()
 
         [[maybe_unused]] auto result = call(job->global_object(), *job, js_undefined());
 
-        // This doesn't match the spec, it actually defines that Job Abstract Closures must return
-        // a normal completion. In reality that's not the case however, and all major engines clear
-        // exceptions when running Promise jobs. See the commit where these two lines were initially
-        // added for a much more detailed explanation.
```

Comment:
Please keep this comment in some form, it's still true & should be noted that we can't assert a normal completion here. The commit I referenced now has a stable hash on the master branch, so if you want to refer to it:
https://github.com/SerenityOS/serenity/commit/a53542e0a3fbd7bf22b685d87f0473e489e1cf42

Reply:
Left the comment with a full hash